docs/manual/cluster-admin/how-to-set-up-virtual-clusters.md

```diff
@@ -82,17 +84,72 @@ hivedscheduler:
 ...
 ```
 
-After modification, use the following commands to apply the settings:
+**We highly recommend you to contain exactly one `skuType` in one virtual cluster.** The support for multiple `skuType` in one VC is not fully implemented. 
```

Comment:
> The support for multiple `skuType` in one VC is not fully implemented.

why

Reply:
I think current implemention is a bit confusing. e.g. Supposing a VC with sku1 (1 GPU 2 CPU) and sku2 (1 GPU 4 CPU), the user cannot submit a 1 GPU 4 CPU job to it because the VC take the minimum CPU number 2 as a requirement. And I remember Fan said we will have a new design of VC in next release. So I change the tone. @yqwang-ms  do you have any comments?

Reply:
We will extend the meaning of skuType in future. At this stage, I suggest to remove this sentence.

Reply:
The orignal issue is related to 'ANY/null skuType'.
But if you only request one specific skuType to a VC with multiple skuTypes, there is no any issue.
So, VC fully supports multiple skuTypes (although we still do not suggest this), and it is the 'ANY/null skuType' cannot proper support.

Reply:
Thanks for the comment. I have removed this sentence.